spec/openhab/core/types/quantity_type_spec.rb

```diff
@@ -11,7 +11,8 @@
     # quantity type operand
     expect(QuantityType.new("50 °F") + QuantityType.new("50 °F")).to eql QuantityType.new("100.0 °F")
     expect(QuantityType.new("50 °F") - QuantityType.new("25 °F")).to eql QuantityType.new("25.0 °F")
-    expect((QuantityType.new("100 °F") / QuantityType.new("2 °F")).to_i).to be 50
+    # @deprecated OH4.0 result of 50. OH 4.1 will result in 1. see https://github.com/openhab/openhab-core/pull/3792
+    expect((QuantityType.new("100 °F") / QuantityType.new("2 °F")).to_i).to be(50).or be(1)
```

Comment:
If I try this with the DSL, I get 1.212272835575194402928498711200641256820715734904441123168785784940181676 as a result. Is it a characteristic of jruby this will round to 1? 1 definitely is not logical.

Reply:
It rounds it to 1 because of the call `to_i` which converts it to integer.

Reply:
Then it is maybe better to select other values as input, as this rounding can create a false impression, and is quite meaningless as a test.

Reply:
Since we're not verifying the actual behavior of relative vs absolute units, I'm just going to avoid using temperature units in this particular test. We only need to verify that multiplication / division can be done without having to verify the implementation details.